command_test.go

```diff
@@ -68,6 +76,7 @@ func TestStripFlags(t *testing.T) {
 	var flagi int
 	var flagstr string
 	var flagbool bool
+	cmdPrint.PersistentFlags().BoolVarP(&flagbool, "persist", "p", false, "help for persistent one")
```

Comment:
Can you add a subcommand of `cmdPrint` with a persistent flag, and verify that stripFlags works?